lib/main.dart

```diff
@@ -1,6 +1,54 @@
 import 'package:flutter/material.dart';
 
-const TextStyle primaryColor = TextStyle(color: Colors.orange);
+const MaterialColor primaryColor = Colors.pink;
+const TextStyle textStylePrimaryColor = TextStyle(color: primaryColor);
+
+Column homepageForm(
+    String stepNumber, String textLabel, String buttonLabel, String hint) {
+  return Column(children: [
+    const Divider(height: 50),
+    Row(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        FittedBox(
+          fit: BoxFit.fitWidth,
+          child: Container(
+            margin: const EdgeInsets.only(right: 50),
+            child: Text(stepNumber,
+                style: const TextStyle(fontSize: 50.0, color: Colors.grey)),
+          ),
+        ),
+        Expanded(
+          child: Column(
+            children: [
+              Align(
+                alignment: Alignment.centerLeft,
+                child: Text(textLabel, style: textStylePrimaryColor),
+              ),
+              TextField(
+                decoration: InputDecoration(
+                  hintText: hint,
+                ),
+              ),
+              Container(
+                margin: const EdgeInsets.only(top: 10),
+                child: TextButton(
+                  style: TextButton.styleFrom(
+                      backgroundColor: primaryColor,
+                      minimumSize: const Size.fromHeight(50)),
+                  onPressed: null,
+                  child: Text(buttonLabel,
+                      style:
+                          const TextStyle(color: Colors.white, fontSize: 15)),
+                ),
+              )
+            ],
+          ),
+        ),
+      ],
+    ),
+  ]);
+}
 
 void main() {
   runApp(const Kilvish());
```

Comment:
Can we have a better class name

Reply:
I would not want to spend too much of time with pre-mature refactoring now. I dont even know what are the best practices of arranging Flutter code.